index.html

```diff
@@ -11,10 +11,11 @@
     <main class="main center">
       <article class="card">
         <img
-          class="card__image"
+          class="card__image-desktop"

```

Comment:
La manera de hacer eso según BEM es usar un Modificador (la M de BE**M**). Es decir,
`class="card__image card__image--desktop"` o esta alternativa que a mi me gusta más (es una versión opinionated mía respecto a BEM):
`class="card__image --desktop"`